docs/source/jupyter/cuds-api.ipynb

```diff
@@ -258,38 +430,178 @@
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "all(n.is_a(city.Neighborhood) for n in c.get(oclass=city.Neighborhood))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "The existing ontology individuals and the relationships among them at the end of the tutorial are depicted below. Note that some attributes that were not specified were set automatically to the default values specified in the ontology."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "- Cuds object named <Freiburg>:\n",
+      "  uuid: 039579cc-ce24-4a74-be89-e24adad58205\n",
+      "  type: city.City\n",
+      "  superclasses: city.City, city.GeographicalPlace, city.PopulatedPlace, cuba.Entity\n",
+      "  values: coordinates: [47  7]\n",
+      "  description: \n",
+      "    To Be Determined\n",
+      "\n",
+      "   |_Relationship city.hasInhabitant:\n",
+      "   | -  city.Citizen cuds object named <Anne>:\n",
+      "   |    uuid: a2e9b4c8-aab5-4354-94d8-b9b9374f1663\n",
+      "   |    age: 25\n",
+      "   |_Relationship city.hasPart:\n",
+      "     -  city.Neighborhood cuds object named <neighborhood 0>:\n",
+      "     .  uuid: 87ae7cd8-be6d-488f-98f1-9b86775ec2d1\n",
+      "     .  coordinates: [0 0]\n",
+      "     -  city.Neighborhood cuds object named <neighborhood 1>:\n",
+      "     .  uuid: ec36bad9-a394-4ae8-a6c3-4402cef8b54f\n",
+      "     .  coordinates: [0 0]\n",
+      "     -  city.Neighborhood cuds object named <neighborhood 2>:\n",
+      "     .  uuid: 7545167d-b53d-4521-9ebb-166acfa8985d\n",
+      "     .  coordinates: [0 0]\n",
+      "     -  city.Neighborhood cuds object named <neighborhood 3>:\n",
+      "     .  uuid: 2bc52e16-eba4-40fb-b1e6-244bb1dae16c\n",
+      "     .  coordinates: [0 0]\n",
+      "     -  city.Neighborhood cuds object named <neighborhood 4>:\n",
+      "     .  uuid: 0e789051-d1be-444f-83cd-00eb042633a0\n",
+      "     .  coordinates: [0 0]\n",
+      "     -  city.Neighborhood cuds object named <neighborhood 5>:\n",
+      "        uuid: fa54102e-2ede-4761-989a-d2825bffc8ba\n",
+      "        coordinates: [0 0]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from osp.core.utils import pretty_print  # This is an utility, see the Utilities section https://simphony.readthedocs.io/en/latest/utils.html?highlight=pretty_print.\n",
```

Comment:
The inline comment is rather long. Might be more readable in a markdown cell

Reply:
Moved to the markdown cell in [6bd48b1](https://github.com/simphony/docs/pull/102/commits/6bd48b18ebc8021526cac643f92dc1ce03c27bb7).